src/response.rs

```diff
@@ -165,6 +165,26 @@ impl Response {
         serde_json::from_reader(self).map_err(::error::from)
     }
 
+    /// Get the response text.
+    ///
+    /// # Example
+    ///
+    /// ```rust
+    /// # extern crate reqwest;
+    /// # fn run() -> Result<(), Box<::std::error::Error>> {
+    /// let content = reqwest::get("http://httpbin.org/range/26")?.text()?;
+    /// # Ok(())
+    /// # }
+    /// ```
+    pub fn text(&mut self) -> io::Result<String> {
+        let len = self.headers().get::<::header::ContentLength>()
+            .map(|ct_len| **ct_len)
+            .unwrap_or(0);
+        let mut content = String::with_capacity(len as usize);
```

Comment:
I feel like we should give the content length an upper bound here just to be a little defensive.

Reply:
emmm.. IMHO this problem should be left to the user, if he/she feels the response body may be large, he/she may want to read the response in another way.

Reply:
Yeh that doesn't seem unreasonable, and might even be common amongst http client implementations. I just don't trust headers for anything :)

Reply:
The `Content-Length` header has to be trusted, or HTTP breaks. hyper will assume any bytes after the length has been reached are not part of this message, and not read them.

As for an upper bound, I dunno. `File::read_to_end` doesn't pick an upper bound, does it?

Reply:
@seanmonstar then... what about the remained question?

Reply:
ping @seanmonstar